hw/ip/rom_ctrl/dv/cov/cover_reg_top.cfg

```diff
@@ -0,0 +1,12 @@
+// Copyright lowRISC contributors (OpenTitan project).
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
++node tb.dut *_tl_*
```

Comment:
It's not really a problem, but isn't this picked up by `hw/dv/tools/vcs/cover_reg_top.cfg`? I think we don't actually need the line?